projects/components/src/file-upload/upload-file-type.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @description
+ * Current set of upload file types
+ *
+ * @usageNotes
+ * This is used to provide supported file types as array into `FileUploadValidators.supportedFileTypes(<>)`
+ *
+ */
+export const enum UploadFileType {
```

Comment:
We should keep this with the validator. Imo, file upload component should not put this restriction in unless we bake in the validation into this component.

Reply:
done